exporter/views.py

```diff
@@ -75,6 +75,9 @@ def do_bagit_export(assets, export_base_dir, export_filename_base):
         text_output_path = os.path.join(dest_path, "%s.txt" % asset_filename)
         with open(text_output_path, "w") as f:
             f.write(asset.latest_transcription or "")
```

Comment:
Is the idea that we would want an empty file for the this-page-is-blank transcriptions?

Reply:
Good question. I'm not sure what the desired behavior is.